.gitmodules

```diff
@@ -0,0 +1,3 @@
+[submodule "grlib"]
+	path = grlib
+	url = https://github.com/suoto/grlib.git
```

Comment:
GRLIB version in use are rather old (GRLIB v1.5.0-b4164, 13 Mar 2016). Latest is grlib-gpl-2020.4-b4261 (released 15 Dec 2020)